ntc_templates/templates/zyxel_os_cfg_static_dhcp_get.textfsm

```diff
@@ -0,0 +1,15 @@
+Value INDEX (\d+)
+Value STATUS (0|1)
+Value MAC_ADDRESS ((([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2})?)
```

Comment:
Similar to others, if this doesn't exist in some cases, need to have tests and likely an alternative strategy.

Reply:
You are right again. It obviously should not be optional